### DISCUSSION TITLE
v0.57.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#2280] [Announcements]
### Enhancements

- The previously used code language is now remembered for subsequent code blocks
- Added the ability to permanently delete documents in trash (#2192)
- Added support for documents written in RTL languages (#2263)
- Collections now have friendlier urls instead of UUID's (#2162)
- Smaller javascript bundle size for faster app loading
- Improved documentation for `SECRET_KEY` generation and local development
- Added optional http logging, add `http` to the list of strings in `DEBUG` env variable to enable (#2183)
- Loading attachments data in document presenter is now parallelized for performance (#2184)
- The document title is now returned in the server HTML when publicly shared for improved unfurls (#2191)
- The "export" permission was removed for read-only users (#2220)
- The templates menu now shows templates from all collections
- Allow embedding of private Mindmeister embeds 
- New languages:
  - 🇧🇷 Brazilian Portugese (#2164)
  - 🇹🇼 Traditional Chinese (#2266)

#### Platform

- Fixed JS error  on old browsers without matchMedia support (#2178)
- Fixed failure to loading collections on frontend results in loading loop (#2176)
- Improved accessibility of sidebar items on mobile (#2043)
- Prevented excessive API request for views data (#2193)
- Fixed incorrect background color on select inputs on non-mac platforms in dark mode (#2188)
- Fixed two restore options visible when an archived document is also deleted (#2194)
- Fixed misalignment of text in templates menu (#2204)
- Fixed copying a link to clipboard no longer also copies dark mode styles (#2218)
- Fixed server error when user cancels OAuth process with Azure SSO (#2231)
- Fixed an issue that could cause documents not to load on the index of large collections (#2254)
- Down arrow in search input now move focus to first result again (#2257)
- Fixed some dropdown menu items now working in Firefox (#2269)

#### Editor

- Changing the type of an existing editor block no longer removes the content (#2156)
- Fixed an issue that could cause incorrect checkboxes to be checked when clicking elsewhere (#2179)
- Fixed an issue where images left or right aligned do not load in publicly shared documents (#2248)

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.57.0'>v0.57.0</a>.</em>

**Comment by dabvastur** [comment]
Is there any kind of problem with version "notification"?, because I pull changes and rebuild docker but the version don't change

![Screenshot_1](https://user-images.githubusercontent.com/48734905/124952733-7bf21e00-dfe2-11eb-93bd-5a9dce307489.png)

**Reply by tommoor** [reply]
Seems good here, it is read from the package json:
https://github.com/outline/outline/blob/v0.57.0/package.json#L210

We recommend using the official docker image:
https://hub.docker.com/repository/docker/outlinewiki/outline
